### PR TITLE
Formatting Fixes

### PR DESCRIPTION
**Pull request by f2prateek**
Changes are described here:

`AdobeIntegration.java`
1. Make comments sentences consistently, before this there was a mix.
2. Use line breaks after statements (for if-else https://google.github.io/styleguide/javaguide.html#s4.3-one-statement-per-line).

`AdobeTest.java`
1. Consistently order qualifiers and annotations on members. See https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations.
2. Consistently use line break after annotations on methods. See https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations.
3. Use line breaks after statements (for if-else https://google.github.io/styleguide/javaguide.html#s4.3-one-statement-per-line).
4. Reformat some lines to fit column width. https://google.github.io/styleguide/javaguide.html#s4.4-column-limit.

`gradle.properties`
1. Group fields for clarity.

### CODE REVIEW

File: src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java [hunk 1]
```diff
@@ -148,25 +148,26 @@ public String key() {
...
-    /** The position of the playhead in seconds when the video was paused */
+    /** The position of the playhead in seconds when the video was paused. */
     long pausedPlayheadPosition;
-    /** The system time in millis at which {@link #pausePlayhead} was invoked */
+    /** The system time in millis at which {@link #pausePlayhead} was invoked. */
     long pauseStartedTime;
     /**
      * The updated playhead position - this variable is assigned to the value a customer passes as
      * properties.seekPosition in a "Video Playback Seek Completed" event or properties.position in
      * a "Video Content Started" event
```
Comment (brennan) [inline src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java]: Should also probably have a period at the end of this line.

File: src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java [hunk 2]
```diff
@@ -85,8 +86,8 @@ public void initialize() {
         .putValue("contextValues", new HashMap<String, Object>())
         .putValue("productIdentifier", "id")
         .putValue("adobeVerboseLogging", true),
-      analytics,
-      Logger.with(VERBOSE),
+        analytics,
+        Logger.with(VERBOSE),
         mockHeartbeatFactory);
```
Comment (brennan) [inline src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java]: Shouldn't `analytics`, `logger.with(VERBOSE)` and `mockHeartbeatFactory` all share a level of indentation that's one less that all the `.putValue`s?

File: src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java [hunk 3]
```diff
@@ -70,7 +70,8 @@ public void setUp() {
     PowerMockito.mockStatic(Analytics.class);
     when(analytics.getApplication()).thenReturn(context);
     integration = new AdobeIntegration(new ValueMap()
-        .putValue("heartbeatTrackingServer", "true"), analytics, Logger.with(NONE), mockHeartbeatFactory);
+        .putValue("heartbeatTrackingServer", "true"), analytics, Logger.with(NONE),
+        mockHeartbeatFactory);
```
Comment (brennan) [inline src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java]: Shouldn't `mockHeartbeatFactory` be indented by one tab fewer?

**Comment by brennan** [comment]
Thanks for taking a look at this @f2prateek - had a few questions on this, but otherwise looks good. I'll review closely all the resources you shared.

**Comment by f2prateek** [comment]
@ladanazita it should have done some of this I think, maybe some of the rules aren't supported yet.

Things it doesn't do:
1. Change comments
2. Non-java files
3. Formatting changes that may change behaviour

**Pull request merged**